### ISSUE TITLE
Add support for worlds and systems to configure join page theme

### ISSUE THREAD

**Issue by mkahvi** [foundryvtt/foundryvtt#9205]
### User Experience

Besides core providing some options for this via #8963, it would be great if game systems could provide their own theme, and if worlds could configure it a bit more themselves (overriding system theme if such exist).

Should also include option to fall back to what Foundry provides, in case the system theme is broken or people just like Foundry's more.

System could provide a special CSS file for this purpose, and worlds could do the same, though I have no idea how to make the world config more user friendly for people who do it for just their own fun.

Default theme should be selected in following order with this:
1) World theme if it exists
2) System theme if it exists and world config doesn't tell to use core theme instead
3) Core Foundry theme

**Comment by dandln** [comment]
Would be good to have support for this extended to just regular modules as well, so login page themes can be shared and used in a secure an easy-to-apply way across different (custom) worlds. Would also make it possible to have dedicated "theme modules".

**Comment by RyanUwU** [comment]
110% Would love this, even if its just to change the foundry icon to something else

**Comment by TMGilchrist** [comment]
Agreed, I would love this and have been wanting this since I first started using Foundry.

**Comment by mkahvi** [comment]
Related to #5234, possibly even duplicate of it, tho I had trouble parsing the other issue for what/where/who of anything.

**Comment by Emilithe-tech** [comment]
Just ran into this today. Foundry applies a 50% opacity to images. This should be able to be set by the system. I have a naturally dark image so adding an opacity filter really reduces the appeal of the image. #5234 would also be amazing. Thay way I can really pull people into the system.
